### PR TITLE
Deep linking docs

### PR DESCRIPTION
**Pull request by human-spider**
# ¡DO NOT MERGE!

Not the final version, for CR and PDF exporting purposes only.

## Demo

http://void-docs.talkable.com/

### CODE REVIEW

File: source/ios_sdk/deep_linking.rst [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+      on your site *Dashboard* at https://admin.talkable.com
+
+7. Add the GetSocial Core framework to your project by following the `GetSocial iOS SDK Manual Integration Guide`_.
+
+.. note::
+
+  You don't need to include or call GetSocial framework explicitly for deep linking to work.
+  The TalkableSDK will detect the GetSocial framework if it's present and will handle deep linking automatically.
+
+.. _`the installation script`: https://talkable-downloads.s3.amazonaws.com/ios-sdk/talkable_ios_sdk_installer.zip
```
Comment (zhuravel) [inline source/ios_sdk/deep_linking.rst]: I propose making the repo public and using GitHub releases for this task, this way you will not have to re-upload the ZIP file to S3 and update the docs each time you update the code on GitHub.
You can just provide this link: `https://github.com/talkable/talkable-ios-sdk-installer/releases/latest`, and create a release on GitHub every time the master is updated.

I use this technique with Magento extension v1 and it has been very convenient.
Magento 1 docs: http://docs.talkable.com/integration/magento/magento1.html#integration-magento-magento1
GitHub releases page: https://github.com/talkable/talkable-magento/releases/latest

Comment (human-spider) [reply source/ios_sdk/deep_linking.rst]: Great idea, thanks

File: source/ios_sdk/deep_linking.rst [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+        <string>tkbl-YOUR_TALKABLE_SITE_ID</string>
+        <string>fb-messenger</string>
+      </array>
+
+    .. note::
+
+      Replace ``YOUR_TALKABLE_SITE_ID`` with your real Talkable Site ID, which can be found
+      on your site *Dashboard* at https://admin.talkable.com
+
+7. Add the GetSocial Core framework to your project by following the `GetSocial iOS SDK Manual Integration Guide`_.
```
Comment (dgtv) [inline source/ios_sdk/deep_linking.rst]: I think that we shouldn't send our customers to Getsocial documentation, as it relies on Getsocial admin panel data to which our customers won't have access.

**Comment by dgtv** [comment]
@human-spider I'm closing it, you can still access the code from this PR (and reopen) or branch if you need

**Pull request closed without merging**